### PR TITLE
Issue #3: added settings button to interface

### PR DESCRIPTION
**Pull request by maximkuzmin**

### CODE REVIEW

File: src/renderer/home/index.tsx [hunk 1]
```diff
@@ -8,6 +8,8 @@ import { H1, Button, ButtonGroup } from '@blueprintjs/core';
 
 import { WindowComponentProps } from 'coulomb/config/renderer';
 
+import {callIPC} from "coulomb/ipc/renderer";
```
Comment (strogonoff) [inline src/renderer/home/index.tsx]: convention: named import syntax separates name from curly braces with a whitespace: `{ callIPC }`.

File: src/renderer/home/index.tsx [hunk 2]
```diff
@@ -66,14 +68,28 @@ const Window: React.FC<WindowComponentProps> = function () {
...
-        <H1 className={styles.appTitle}>Glossarist</H1>
+          <div className={styles.headerAndSettings}>
+            <H1 className={styles.appTitle}>Glossarist</H1>
+            <Button
+              icon="settings"
+              title="Settings"
+              onClick={openSettingsWindow}
+              className={styles.settingsButton}
+              minimal={true}
+              />
```
Comment (strogonoff) [inline src/renderer/home/index.tsx]: Convention: dedent `/>` for self-closing tags
 
```
            <Button
               icon="settings"
               title="Settings"
             />
```

File: src/renderer/home/styles.scss [hunk 3]
```diff
@@ -110,8 +110,20 @@ html, body, #app {
...
 
+  .headerAndSettings {
+     display: flex;
+     align-items: baseline;
+     flex-direction: row;
+     justify-content: flex-end;
+   }
+
+   .settingsButton {
+     margin-left: 0.3 rem;
```
Comment (strogonoff) [inline src/renderer/home/styles.scss]: Convention: no space between the value and the unit in numeric CSS property values: `margin-left: 0.3rem;`

File: src/renderer/home/index.tsx (hunk 1 above)
Comment (strogonoff) [inline src/renderer/home/index.tsx]: Convention: Use single quotess in import paths

**Comment by strogonoff** [comment]
Fixed, merging

**Comment by strogonoff** [comment]
@ronaldtse For the recordm, I believe build failures are due to how our GHA setup works with pull requests/forks/branches. Diff looks good and no compile errors were encountered.

**Comment by ronaldtse** [comment]
The merged build was successful. Let me check.

**Comment by ronaldtse** [comment]
Strange, the only difference in the PR build and master build is:
```
  • empty password will be used for code signing  reason=CSC_KEY_PASSWORD is not defined
146
```

But the secret is set in GH settings. Not sure why the behavior differs.

**Comment by ronaldtse** [comment]
Actually this turns out to be a security feature so that secrets are not shared :
https://github.community/t5/GitHub-Actions/Allow-secrets-to-be-shared-with-trusted-Actions/td-p/34278

So we will need a mechanism to disable code-signing in the build process that does not rely on secrets.

**Comment by strogonoff** [comment]
Confirmed, CI passes after merge

**Pull request merged**